scripts/migrate.py

```diff
@@ -283,5 +313,22 @@ def _batch_iter(n, iterable):
         yield batch
 
 
+def collect_es_annotation_ids(es):
+    ids = set()
+    annotations = scan(es)
+    for batch in _batch_iter(BATCH_SIZE, annotations):
+        for ann in batch:
+            ids.add(ann['_id'])
+
+    return ids
+
+
+def collect_pg_annotations():
+   ids = set()
```

Comment:
You're collecting them all in one big set anyway, so I `yield_per` doesn't buy you anything. I suspect you previously had this function returning a set of ids, no?